Content.Shared/Chemistry/EntitySystems/ReagentDispenserSystem.cs

```diff
@@ -0,0 +1,208 @@
+using System.Linq;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.FixedPoint;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Labels.Components;
+using Content.Shared.Nutrition.EntitySystems;
+using Content.Shared.Storage;
+using Content.Shared.Storage.EntitySystems;
+using JetBrains.Annotations;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Containers;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Chemistry.EntitySystems;
+
+/// <summary>
+/// Contains all the server-side logic for reagent dispensers.
```

Comment:
server-side? :P

Reply:
Yeah it means like. Service side. Totally intentional, 'cause it's used... in the service department. :)